src/world/map.c

```diff
@@ -3754,11 +3749,12 @@ void game_command_place_large_scenery(int* eax, int* ebx, int* ecx, int* edx, in
 		}
 	}
 
-
+	uint32 num_elements = 0;
```

Comment:
well its not consistent in the C files but the main reason is because i copied and pasted the variable name from the function description.